myLivingDead/src/main/java/Entity/Survivor/Type/Child.java

```diff
@@ -0,0 +1,28 @@
+package Entity.Survivor.Type;
+
+import Entity.Entity;
+import Entity.Survivor.Survivor;
+
+public class Child extends Survivor {
+    private int health = 20;
+    private int damage = 2;
```

Comment:
Would it make sense to make this final for each Type class because the damage will always be constant?

Reply:
I think this does make sense